test/e2e/data/infrastructure-metal3/cluster-template-ubuntu/kustomization.yaml

```diff
@@ -0,0 +1,7 @@
+bases:
+- ../bases/cluster-with-kcp.yaml
+- ../bases/ippool.yaml
+- ../bases/md.yaml
```

Comment:
Same here as in the other `kustomize.yaml`.

Reply:
see the comment